config/src/test/java/io/quarkus/ts/configmap/api/server/OpenShiftFailOnMissingConfigIT.java

```diff
@@ -21,6 +20,7 @@ public class OpenShiftFailOnMissingConfigIT {
             .withProperty("quarkus.kubernetes-config.config-maps", "absent-config-map");
 
     @Test
+    @Disabled("https://github.com/quarkusio/quarkus/issues/38481")
```

Comment:
https://github.com/quarkusio/quarkus/issues/38481 is closed, we don't need to resolve it now, but it would be nice to add comment to for others to understand what is that status and what's next

Reply:
Added a comment